### ISSUE TITLE
Gutenberg Editor Showing Instead Of "Edit With Elementor"

### ISSUE THREAD

**Issue by mdrideout** [elementor/elementor#8647]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
When I created a new page and click "Edit with Elementor", the elementor editor loads. When I go back to "view dashboard", or just the regular "edit" screen for a page, the content is displayed in the gutenberg editor or classic editor instead of that being overridden by the "edit with elementor" overlay. 

This is new to the latest update to elementor I recently performed on multiple sites.

This is a problem for people that may think they should edit the content in the gutenberg editor instead of proceeding to elementor. 

## Steps to reproduce
See above.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.14.0
	MySQL version: Source distribution v5.6.31
	PHP Version: 7.1.28
	PHP Max Input Vars: 1000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.2.2
	Site URL: https://swansongroup.biz
	Home URL: https://swansongroup.biz
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 1.0.0
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 1.8.7
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.142 Safari/537.36

== Active Plugins ==
	Astra Pro
		Version: 1.8.7
		Author: Brainstorm Force

	Better Search Replace
		Version: 1.3.3
		Author: Delicious Brains

	Elementor
		Version: 2.6.5
		Author: Elementor.com

	Elementor Pro
		Version: 2.5.14
		Author: Elementor.com

	Google Analytics for WordPress by MonsterInsights
		Version: 7.7.1
		Author: MonsterInsights

	Imsanity
		Version: 2.4.2
		Author: Exactly WWW

	Premium Addons for Elementor
		Version: 3.5.7
		Author: Leap13

	Premium Addons PRO
		Version: 1.6.1
		Author: Leap13

	Premium Blocks for Gutenberg
		Version: 1.7.2
		Author: Leap13

	Proxy Cache Purge
		Version: 4.8.1
		Author: Mika Epstein

	Really Simple SSL
		Version: 3.2.3
		Author: Rogier Lankhorst, Mark Wolters

	Safe SVG
		Version: 1.9.4
		Author: Daryll Doyle

	Ultimate Addons for Elementor
		Version: 1.14.0
		Author: Brainstorm Force

	Ultimate Addons for Gutenberg
		Version: 1.13.6
		Author: Brainstorm Force

	Wordfence Security
		Version: 7.3.5
		Author: Wordfence

	Yoast SEO
		Version: 11.6
		Author: Team Yoast


== Log ==
	: 
Log: showing 20 of 322019-04-05 20:46:44 [info] Elementor Pro/Upgrades - _v_2_5_0_popup_border_radius Finished 
2019-04-05 20:46:44 [info] Elementor Pro/Upgrades - _v_2_5_4_form Start  
2019-04-05 20:46:44 [info] Elementor Pro/Upgrades - _v_2_5_4_form Finished 
2019-04-05 20:46:44 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.8',
  'to' => '2.5.4',
)]
2019-04-10 16:12:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.11',
  'to' => '2.5.13',
)]
2019-04-10 16:14:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.5.4',
  'to' => '2.5.5',
)]
2019-04-24 21:40:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.13',
  'to' => '2.5.14',
)]
2019-05-27 22:26:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.14',
  'to' => '2.5.15',
)]
2019-05-27 22:26:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.5.5',
  'to' => '2.5.8',
)]
2019-06-14 21:11:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.15',
  'to' => '2.5.16',
)]
2019-06-14 21:11:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.5.8',
  'to' => '2.5.9',
)]
2019-07-12 17:33:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.5.9',
  'to' => '2.5.13',
)]
2019-07-12 17:33:31 [info] elementor::elementor_updater Started 
2019-07-12 17:33:31 [info] Elementor/Upgrades - _v_2_6_0_fa4_migration_flag Start  
2019-07-12 17:33:31 [info] Elementor/Upgrades - _v_2_6_0_fa4_migration_flag Finished 
2019-07-12 17:33:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.16',
  'to' => '2.6.1',
)]
2019-07-12 17:33:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.16',
  'to' => '2.6.1',
)]
2019-07-15 20:24:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.6.1',
  'to' => '2.6.2',
)]
2019-07-15 20:24:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.5.13',
  'to' => '2.5.14',
)]
2019-07-19 16:46:59 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.6.2',
  'to' => '2.6.5',
)]

JS: showing 10 of 10JS: 2019-04-06 02:48:08 [error X 4][https://dev.rideout.us/swanson/wp-admin/js/editor.min.js?ver=5.1.1:1:694] Cannot read property \'style\' of undefined 
JS: 2019-04-06 02:48:08 [error X 4][https://dev.rideout.us/swanson/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.11:2:215529] Cannot read property \'on\' of null 
JS: 2019-04-06 23:05:59 [error X 1][https://dev.rideout.us/swanson/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.11:2:8507] Cannot read property \'select2\' of undefined 
JS: 2019-06-21 11:47:02 [error X 30][https://dev.rideout.us/swanson/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.16:2:27759] Cannot read property \'get\' of undefined 
JS: 2019-06-26 15:14:45 [error X 52][https://dev.rideout.us/swanson/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.16:2:78958] Object doesn\'t support property or method \'contains\' 
JS: 2019-06-26 19:51:02 [error X 4][https://dev.rideout.us/swanson/wp-admin/load-scripts.php?c=0&load%5B%5D=underscore,jquery-core,jquery-migrate,backbone,shortcode,utils,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-plupload,jqu&load%5B%5D=ery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=5.2.2:1:15147] Syntax error 
JS: 2019-06-26 19:51:45 [error X 1][https://dev.rideout.us/swanson/wp-admin/js/editor.min.js?ver=5.2.2:1:681] Unable to get property \'style\' of undefined or null reference 
JS: 2019-06-26 19:51:45 [error X 1][https://dev.rideout.us/swanson/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.16:2:215835] Unable to get property \'on\' of undefined or null reference 
JS: 2019-06-26 19:52:22 [error X 2][https://dev.rideout.us/swanson/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.16:2:209127] Unable to get property \'0\' of undefined or null reference 
JS: 2019-06-26 20:09:37 [error X 12][:1:3254] Object doesn\'t support property or method \'values\' 

PHP: showing 1 of 1PHP: 2019-04-24 21:53:20 [notice X 4][/home/dev.rideout.us/public_html/swanson/wp-content/plugins/elementor-pro/modules/query-control/module.php::280] Undefined index: object_type [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by shilo-ey** [comment]
Thanks for reporting.

This issue has been already taken care of and soon we will release a patch.

ref: #8628 

Thanks

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v2.6.6

Feel free to update

Thanks!
